src/FrameReflower/Image.php

```diff
@@ -115,7 +117,7 @@ function get_min_max_width()
             $style->max_height !== "none"
         ) {
 
-            list($w, $h) = $this->_frame->get_containing_block();
+            list($w, $h) = $frame->get_containing_block();
```

Comment:
In #2108 this was changed from:
```
list( /*$x*/, /*$y*/, $w, $h) = $this->_frame->get_containing_block();
```
to:
```
list($w, $h) = $this->_frame->get_containing_block();
```

Does the list portion need to be `list(, , $w, $h)` or `list('w' => $w, 'h' => $h)` instead?

Reply:
This indeed should be a bug for that case. I'll fix that tomorrow and have a look why the tests fail to cover that.

Reply:
I fixed it with reverting the change. The tests are not good enough in this area to cover it. My change not make a difference. This is caused by too many necessary mock objects returning a new value inside the method.. It could be improved though with a lot of `shouldReceive->with` parameter checks.
On the other hand it could be a better effort to improve the structure of those classes.